packages/react-diagrams-core/package.json

```diff
@@ -37,5 +37,8 @@
 		"lodash": "4.*",
 		"react": "16.*"
 	},
-	"gitHead": "bb878657ba0c2f81764f32901fd96158a0f8352e"
+	"gitHead": "bb878657ba0c2f81764f32901fd96158a0f8352e",
+	"devDependencies": {
+		"resize-observer-polyfill": "^1.5.1"
```

Comment:
this needs to be moved to a peer dependency and added as a dev dependency on the root package

Reply:
Done.